footer.css

```diff
@@ -0,0 +1,29 @@
+.footer-terms{
+   background: #f2f2f2;
+   border-top: 1px solid #dcdee0;
+   margin-top: 25px;
+   padding: 20px 50px;
+}  
+.footer-links a{
+   text-decoration:none;
+   color:#b8a056;
+   margin:0 10px;
+}
+.footer-logo{
+   display: flex;
+   justify-content: space-between;
+}
+.tipser-logo img{
+  width: 120px;
+}
+.footer-links{
+    font-size: 0.8em;
+    text-align: center;
+}
+.footer-link{
+   display: inline-block;
+}
+.footer-copyrights{
+   text-align: center;
+   margin-top:20px;
+}
```

Comment:
Empty line is needed at the end of every code file

Reply:
done